src/components/forms/inputFieldOnly.vue

```diff
@@ -0,0 +1,280 @@
+				if (this.input == null || this.input == '') {
+					this.errMsgs.push(this.label + " is required");
+					return true;
+				}
+			}
+			return false
+		},
+		stringMinInvalid: function() {
+			if (this.minLength) {
+				if (this.input.length < this.minLength) {
+					this.errMsgs.push(this.label + " must be at least " + this.minLength.toString() + " characters");
+					return true;
+				}
+			}
+			return false;
+		},
+		stringMaxInvalid: function() {
+			if (this.maxLength) {
+				if (this.input.length > this.maxLength) {
+					this.errMsgs.push(this.label + " must be less than " + this.maxLength.toString() + " characters");
+					return true;
+				}
+			}
+			return false;
+		},
+		numMinInvalid: function() {
+			if (this.minNum != null) {
+				if (this.input < this.minNum) {
+					this.errMsgs.push(this.label + " must be greater than " + this.minNum.toString());
+					return true;
+				}
+			}
+			return false;
+		},
+		numMaxInvalid: function() {
+			if (this.maxNum) {
+				if (this.input > this.maxNum) {
+					this.errMsgs.push(this.label + " must be less than " + this.maxLength.toString());
+					return true;
+				}
+			}
+			return false;
+		},
+		dateMinInvalid: function() {
+			if (this.minDate) {
+				if (this.input < this.minDate) {
+					this.errMsgs.push(this.label + " must be greater than " + this.minDate.toString());
+					return true;
+				}
+			}
+			return false;
+		},
+		dateMaxInvalid: function() {
+			if (this.maxDate) {
+				if (this.input > this.maxDate) {
+					this.errMsgs.push(this.label + " must be less than " + this.maxDate.toString());
+					return true;
+				}
+			}
+			return false;
+		},
+
+		runValidators: function() {
+			this.errMsgs = [];
+			let validators = [
+				this.requiredInvalid,
+				this.stringMinInvalid,
+				this.stringMaxInvalid,
+				this.numMinInvalid,
+				this.numMaxInvalid,
+				this.dateMinInvalid,
+				this.dateMaxInvalid
+			]
+			let invalid = false;
+			validators.forEach(v => {
+				if (v()) { invalid = true; }
+			});
+			return invalid;
+		},
+
+		validateInput: function() {
+			if (this.runValidators()) {
+					this.invalid = true;
+				}
+			else {
+				this.invalid = false;
+				console.log("Validate", this.invalid);
+			}
+		},
+
+		escapeChars (str) {
+			str = str.replace(/</g, '&lt;');
+			return str.replace(/>/g, '&gt;');
+		},
+
+        isCheckbox: function() {
+            if (this.type == 'checkbox') {
+                return true;
+            }
+            else {
+                return false;
+            }
+        },
+
+	},
+	computed: {
+		inValEscaped: {
+			get () {
+				return this.input;
+			},
+			set (value) {
+				if (typeof(value) == "string") {
+					this.input = this.escapeChars(value);
+				}
+				else {
+					this.input = value;
+				}
+			}
+		},
+	},
+}
+</script>
+
+<style scoped>
+  .checkbox {
+    position: relative;
+    display: inline-block;
+    border: 1px solid #a9a9a9;
+    border-radius: .25em;
+    width: 1.4em;
```

Comment:
this is causing an odd shadow to the right of your checkboxes. If you make the width auto and the height pixels, then it fixes it

Reply:
I take that back, it is weird. you could have a width of 1em and keep your height of 1.4em to get your current size working

Reply:
sounds good. done.